contracts/evoting/controller/action.go

```diff
@@ -44,7 +44,15 @@ import (
 )
 
 const contentType = "application/json"
+const electionPath = "/evoting/elections"
+const electionPathSlash = "/evoting/elections/"
+const electionIDPath = "/evoting/elections/{electionID}"
+const UNEXPECTED_STATUS_STR = "unexpected status: %s, body: %s"
+const FAIL_RETRIEVE_DECRYPTION_STR = "failed to retrieve decryption key: %v"
+const SELECT_STR = "select:"
 const getElectionErr = "failed to get election: %v"
+const FAILED_CAST_STR = "failed to cast vote: %v"
+const RESP_BODY_STR = "response body: "
```

Comment:
Same here about the naming convention. You can also change the existing constant for consistency (eg. `errGetElection` vs `getElectionErr`).